site/en/docs/extensions/migrating/mv3-status/index.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Manifest V3 Status
+description: "Known issues when migrating Chrome Extensions to Manifest V3."
+date: 2022-09-23
+updated: 2022-04-13
+tags:
+  - extensions-news
+---
+
+The Manifest V2 deprecation is on hold. In the meantime, we're improving the platform and documentation to ensure the transition from Manifest V2 to Manifest V3 is smooth and successful. Watch this page for our progress. We'll provide sufficient migration time for developers&mdash;at least 6 months&mdash;before beginning any experiments to turn off MV2.
+
+## Planned enhancements {: #planned-enhancements }
+
+The following enhancement is currently under development.
+
+### Userscript managers support {: #userscript-managers-support }
```

Comment:
Suggestion: use bullet points similar to the completed work below

Reply:
My guess is that readers of this page are more interested in what MV3 can't do than in what it can. They're asking either 'can I convert my MV2 extension now?' or 'does an (MV3 extension) solve my use case?'. As a courtesy I thought I would save them from yo-yoing between pages.